geoindicators/src/main/groovy/org/orbisgis/BlockIndicators.groovy

```diff
@@ -0,0 +1,42 @@
+package org.orbisgis
+
+import groovy.transform.BaseScript
+import org.orbisgis.datamanager.JdbcDataSource
+import org.orbisgis.datamanagerapi.dataset.ITable
+import org.orbisgis.processmanagerapi.IProcess
+
+@BaseScript Geoclimate geoclimate
+
+/**
+ * This process is used to compute basic statistical operations on a specific variable from a lower scale (for
+ * example the sum of each building volume constituting a block to calculate the block volume)
+ * @return A database table name.
+ * @author Erwan Bocher
+ */
+static IProcess unweightedOperationFromLowerScale() {
```

Comment:
aggregatedStatistics ?

Reply:
unweightedAggregatedStatistics in order to differentiate from the weighted ones ?